tests/integrations/mcs/scheduling/server_test.go

```diff
@@ -261,6 +261,8 @@ func (suite *serverTestSuite) TestDisableSchedulingServiceFallback() {
 
 	// API server will execute scheduling jobs since there is no scheduling server.
 	testutil.Eventually(re, func() bool {
+		re.NotNil(suite.pdLeader.GetServer())
+		re.NotNil(suite.pdLeader.GetServer().GetRaftCluster())
 		return suite.pdLeader.GetServer().GetRaftCluster().IsSchedulingControllerRunning()
```

Comment:
how about `return suite.pdLeader.GetServer()!=nil && suite.pdLeader.GetServer().GetRaftCluster()!=nil && suite.pdLeader.GetServer().GetRaftCluster().IsSchedulingControllerRunning()`

Reply:
I just want to see who is nil.

Reply:
`re.NotNil` also will make test unstable

Reply:
how about change titile to `test: add more info for TestDisableSchedulingServiceFallback`?